dev/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:20.04
+SHELL ["/bin/bash", "-c"]
+
+# tools
+RUN su - \
+    && apt update \
+    && apt-get install sudo -y \
+    && apt -y install \
+                curl \
+                git \
+                nano \
+                g++ \
+                python \
+                make \
+                postgresql-server-dev-12 \
+                zip \
+                unzip
+
+# install docker (inception)
+RUN curl -fsSL https://get.docker.com -o get-docker.sh && \
+    sudo sh get-docker.sh
```

Comment:
sudo needed?

Reply:
Naively taken from [docs](https://docs.docker.com/engine/install/ubuntu/#install-using-the-convenience-script). Most likely not needed.

Thinking things over, since currently, neither Docker nor Docker Compose are needed, and since there are other problems with such inception, the whole thing was removed. 

Re-review will be needed (repo setting nullify reviews on code change...).